cmd/juju/machine/upgrademachine.go

```diff
@@ -287,28 +305,33 @@ func (c *upgradeSeriesCommand) UpgradeSeriesPrepare(ctx *cmd.Context) (err error
 		return errors.NotFoundf("units for machine %q", c.machineNumber)
 	}
 
-	if err := c.promptConfirmation(ctx, units); err != nil {
+	s, err := series.GetSeriesFromBase(base)
+	if err != nil {
+		return errors.Annotatef(err, "invalid series base")
```

Comment:
what is a "series base"?

Reply:
This is just temporary. It's an error message about converting a base from a series. I might just re-word this to `"can not convert base to series"`.